Custom Property Update

kasperlaurberg opened the pull request:
Added text about language independent custom properties & language dependent custom properties
Added text about customer property templates
Added/updated examples

Still to do not by KBL:
Update the how to sections
Updates images
Add additional images
Fix my mistakes

Chatrine commented:
Test

kasperlaurberg commented on docs/06-Working with Data/03-Customisable Data/custom-properties.mdx:
```diff
@@ -9,14 +10,32 @@ slug: /custom-properties
+ - The calendar id of a room used for booking
+ - Ids of a location in other systems
+
+If  a value is added to the same Key on both a Language independent Custom Property and Language dependent Custom Property, the Key-Value pair on the specific language overrides the generic Custom Property.
```
@Chatrine What would you like change about this segement?

kasperlaurberg replied:
@Chatrine A possible addition to the current text: 

> If a Key exists as both a Language independent Custom Property and Language dependent Custom Property, but a value is added for only the Language independent Custom Property then that value will be send to the SDKs regardless of the users preferred language.

Chatrine replied:
@kasperlaurberg I want to add something about another scenario as well. It could be something similar to the following (but maybe polished a bit): 

"If a value is added to the same Key on both a Language independent Custom Property and Language dependent Custom Property on the default language. But there are more than one language, these other languages will inherit the value from the Language independent Custom Property, unless something else is added on each specific language."

Chatrine replied:
@kasperlaurberg We should probably also include your suggestion, just to cover all cases of special behavior 😃

kasperlaurberg commented:
@Chatrine I added some additional explanation to make everything more clear. I also changed the naming to Language Specific & Generic Custom Properties to make the difference more clear

kasperlaurberg commented on docs/06-Working with Data/03-Customisable Data/custom-properties.mdx:
```diff
@@ -10,27 +10,45 @@ slug: /custom-properties
+|----------|:-----------------:|:-------:|:---------:|
+| 1        |         A         |         |     A     |
+| 2        |                   |    B    |     B     |
+| 3        |         A         |    B    |     B     |
```
Yes. But only because I swapped the columns when made the name change.
I have swapped them back. Good catch
